lib/chat/user_message_rows.dart

```diff
@@ -73,6 +59,7 @@ buildMessageRows(
     var lastMessage = match.lastMessage;
     var lastMessageType = lastMessage.type;
     var isSent = lastMessage.idFrom == id;
+    var isRead = !isSent && lastMessage.isRead == true;
```

Comment:
Do you need the `== true` part? I think it's automatically interpreted as a boolean with the `&&` operator.